test/ctest_helpers.cmake

```diff
@@ -116,6 +116,8 @@ function(create_test_executable target)
         )
     endif()
 
+    target_link_libraries(${target} m)
```

Comment:
This is an annoying side effect, from now on every test compiled under the function `create_test_executable` will be linked to the math library. A workaround could be implementing pow directly.